python/sglang/srt/utils.py

```diff
@@ -2105,6 +2106,25 @@ def get_free_port():
             return s.getsockname()[1]
 
 
+def get_local_ip_by_nic(interface: str) -> str:
```

Comment:
nit: would be great to unify this with get_local_ip_by_remote to avoid copy-pasting